tasks/lint.yaml

```diff
@@ -11,3 +15,11 @@ tasks:
       # requires yamllint to be installed either from pip or brew
       - description: Lint YAML files
         cmd: yamllint . -c .yamllint --no-warnings
+
+  - name: oscal
+    description: Run linting checks on OSCAL
+    actions:
+      - description: Lint OSCAL files
+        cmd: lula tools lint -f ${OSCALFILES}
+        setVariables:
```

Comment:
reading on [variables](https://github.com/defenseunicorns/maru-runner?tab=readme-ov-file#variables) I do not believe you need this `setVariables` definition.